bcda/auth/okta.go

```diff
@@ -76,15 +76,21 @@ func (o OktaAuthPlugin) RevokeClientCredentials(params []byte) error {
 }
 
 func (o OktaAuthPlugin) RequestAccessToken(creds Credentials, ttl int) (Token, error) {
-	if creds.ClientID == "" {
+	clientID := creds.ClientID
+	// Also accept clientID via creds.UserID to match alpha auth implementation
```

Comment:
conversely, i think this comment makes sense to have.  it's obvious _how_ the code works but we don't know _why_ we're doing it without the comment. 👍

Reply:
👍